lib/Backend/IRBuilder.cpp

```diff
@@ -750,28 +751,41 @@ IRBuilder::Build()
             }
         }
 #endif
-        if (IsLoopBodyInTry() && m_lastInstr->GetDst() && m_lastInstr->GetDst()->IsRegOpnd() && m_lastInstr->GetDst()->GetStackSym()->HasByteCodeRegSlot())
+
+        if (IsLoopBodyInTry() && lastProcessedInstrForJITLoopBody != m_lastInstr)
```

Comment:
Yes, I hit a case that one loop iteration doesn't generate any new IR instr, then added this check to make it general.

---
In reply to: [100933833](https://github.com/Microsoft/ChakraCore/pull/2518#discussion_r100933833) [](ancestors = 100933833)